packages/server/lib/plugins/child/require_async_child.js

```diff
@@ -1,5 +1,16 @@
 process.title = 'Cypress: Config Manager'
 
+const { telemetry, OTLPTraceExporterIpc, decodeTelemetryContext } = require('@packages/telemetry')
+
+const { file, projectRoot, telemetryCtx } = require('minimist')(process.argv.slice(2))
+
+const { context, version } = decodeTelemetryContext(telemetryCtx)
```

Comment:
In the case `telemetryCtx` isn't defined in `decodeTelemetryContext`, would `context` and `version` being undefined here cause issues with the telemetry initialization? Is it better for `decodeTelemetryContext` to throw if no context is given?

Reply:
I don't want to throw, but i can check for context and version before initializing telemetry